package.json

```diff
@@ -117,7 +117,7 @@
     "build-docs": "documentation build --github --format html --config documentation.yml --theme ./docs/_theme --output docs/api/",
     "build": "npm run build-docs # invoked by publisher when publishing docs on the mb-pages branch",
     "start-docs": "npm run build-min && npm run build-docs && jekyll serve --watch",
-    "lint": "eslint --ignore-path .gitignore js test bench docs/_posts/examples/*.html debug/*.html",
+    "lint": "eslint --fix --ignore-path .gitignore js test bench docs/_posts/examples/*.html debug/*.html",
```

Comment:
I recently learned that you can do this with npm:

```
npm run lint -- --fix
```

.. and it'll pass the `--fix` flag down to the `eslint` command 😄